hpack/shared/src/test/scala/org/http4s/hpack/TestCase.scala

```diff
@@ -129,14 +129,14 @@ case class TestCase(
 
   private def createEncoder() =
     new Encoder(
-      maxHeaderTableSize.getOrElse(Int.MaxValue),
+      maxHeaderTableSize.getOrElse(4096),
       useIndexing.getOrElse(true),
       forceHuffmanOn.getOrElse(false),
       forceHuffmanOff.getOrElse(false),
     );
 
   private def createDecoder() =
-    new Decoder(8192, maxHeaderTableSize.getOrElse(Int.MaxValue));
+    new Decoder(8192, maxHeaderTableSize.getOrElse(4096));
```

Comment:
The `Int.MaxValue` default when no `maxHeaderTableSize` was specified in the test case leads to massive allocations which degraded performance on Node.js. Since this is only test code it's probably ok to use 4096 which is the default in Ember.

Thanks to @ChristopherDavenport for helping to chase this down.